boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp.yaml

```diff
@@ -10,6 +10,6 @@ toolchain:
   - xtools
   - zephyr
 ram: 256
-flash: 512
+flash: 600
```

Comment:
Why is this modified? `cpuapp_slot0_partition` size is still 512 kB.

Reply:
Isn't it size of flash used ? @57300 asked to update it.

Reply:
No, it's the available flash. See [Board configuration](https://docs.zephyrproject.org/latest/develop/test/twister.html#board-configuration).

Reply:
Ok then I'll revert it

Reply:
done